src/index.css

```
@@ -140,6 +147,21 @@ All colors MUST be HSL.
 
   body {
     @apply bg-background text-foreground;
-    font-family: 'Vesper Libre', serif;
+    font-family: 'Roboto', 'Segoe UI', 'Vesper Libre', sans-serif;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)